demo-projects/blog/index.js

```diff
@@ -18,7 +19,11 @@ const keystone = new Keystone({
     const users = await keystone.lists.User.adapter.findAll();
     if (!users.length) {
       const initialData = require('./initialData');
-      await keystone.createItems(initialData);
+      await createItems({
+        keystone,
+        listName: 'User',
+        items: initialData.User.map(x => ({ data: x })),
```

Comment:
Let's update the data in the `initialData` file to have the `{ data: ... }` key so that we can keep this line of code a bit cleaner.